src/main/webapp/app/course/plagiarism-cases/instructor-view/detail-view/plagiarism-case-instructor-detail-view.component.ts

```diff
@@ -188,7 +220,7 @@ export class PlagiarismCaseInstructorDetailViewComponent implements OnInit, OnDe
     /**
      * exports the plagiarism case with all relevant information as PDF
      */
-    exportPlagiarismCase(): void {
-        // TODO: export the plagiarism case as a PDF with all relevant information
+    printPlagiarismCase(): void {
+        window.print();
```

Comment:
Please use the [print method of the theme service](https://github.com/ls1intum/Artemis/blob/13149b440006186c949c132a307dd9c2d3332ff6/src/main/webapp/app/core/theme/theme.service.ts#L159) instead as that will temporarily remove the dark mode for printing.